src/main/c/cext/call.c

```diff
@@ -214,3 +214,7 @@ VALUE rb_eval_cmd_kw(VALUE cmd, VALUE args, int kw_splat) {
     return RUBY_CEXT_INVOKE("rb_eval_string", cmd);
   }
 }
+
+int rb_frame_method_id_and_class(ID *idp, VALUE *klassp) {
+    return RUBY_CEXT_INVOKE("rb_frame_method_id_and_class", idp, klassp);
+}
```

Comment:
We would need specs for this, under `spec/ruby/optional/capi`.
`rb_frame_method_id_and_class` is actually not a private function but it seems public C API, it's declared and documented in https://github.com/ruby/ruby/blob/fc48a679062e422b475ccea11574bf5b4368b732/include/ruby/internal/intern/vm.h#L62